src/main/java/edu/kis/powp/jobs2d/drivers/CurrentDriverWrapper.java

```diff
@@ -0,0 +1,20 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.AbstractDriver;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DriverFeature;
+
+public class CurrentDriverWrapper extends AbstractDriver {
```

Comment:
The name does not fully indicate the purpose of the adapter or what was actually adapted.